controllers/utils/volumegroup.go

```diff
@@ -19,17 +19,47 @@ package utils
 import (
 	"context"
 	"fmt"
+	"k8s.io/apimachinery/pkg/types"
 
 	volumegroupv1 "github.com/IBM/csi-volume-group-operator/api/v1"
 	"github.com/IBM/csi-volume-group-operator/pkg/messages"
 	"github.com/go-logr/logr"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/util/retry"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+func GetVG(client client.Client, logger logr.Logger, vgName string, vgNamespace string) (*volumegroupv1.VolumeGroup, error) {
+	logger.Info(fmt.Sprintf(messages.GetVGC, vgName, vgNamespace))
+	vgc := &volumegroupv1.VolumeGroup{}
+	namespacedVGC := types.NamespacedName{Name: vgName, Namespace: vgNamespace}
+	err := client.Get(context.TODO(), namespacedVGC, vgc)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			logger.Error(err, "VolumeGroupContent not found", "VolumeGroupContent Name", vgName)
+		}
+		return nil, err
+	}
+
+	return vgc, nil
+}
```

Comment:
```suggestion
func GetVG(client client.Client, logger logr.Logger, vgName string, vgNamespace string) (*volumegroupv1.VolumeGroup, error) {
	logger.Info(fmt.Sprintf(messages.GetVG, vgName, vgNamespace))
	vg := &volumegroupv1.VolumeGroup{}
	namespacedVG := types.NamespacedName{Name: vgName, Namespace: vgNamespace}
	err := client.Get(context.TODO(), namespacedVG, vg)
	if err != nil {
		if errors.IsNotFound(err) {
			logger.Error(err, "VolumeGroup not found", "VolumeGroup Name", vgName)
		}
		return nil, err
	}

	return vg, nil
}
```
and we don't have `messages.GetVG` so we need to create one

Reply:
done